### PR TITLE
Add release workflow

### PR DESCRIPTION
**Pull request by sdclarke**
Add a workflow which runs when new tags are created to push the role to Ansible Galaxy

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        with:
+          python-version: '3.x'
+      - name: Install Ansible.
+        run: pip3 install ansible-core==2.17.*
+      - name: Start import on Ansible Galaxy
+        run: >
+          ansible-galaxy role import
+          --token ${{ secrets.ANSIBLE_GALAXY_API_TOKEN }}
+          $(echo ${{ github.repository }} | cut -d/ -f1)
+          $(echo ${{ github.repository }} | cut -d/ -f2)
```
Comment (jjardon) [inline .github/workflows/release.yaml:23]: @sdclarke This looks good to me

Let's add a CI job to check the ansible correctness (so it runs when a PR is sent) (something like `ansible-playbook --syntax-check` or similar)

**Review by jjardon — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (sdclarke) [reply .github/workflows/release.yaml:23]: Added a job to run ansible-lint on pull requests.

**Review by jjardon — CHANGES_REQUESTED** [review]
Split the "Add linting to CI and fix linting failures" commit in 2 please: one to fix the problems, other to add the linting job

**Comment by sdclarke** [comment]
> Split the "Add linting to CI and fix linting failures" commit in 2 please: one to fix the problems, other to add the linting job

Done

**Review by jjardon — APPROVED** [review]

**Pull request merged**